doc/nrf/device_guides/wifi_coex.rst

```diff
@@ -242,6 +242,71 @@ To enable the generic three-wire coexistence, do the following:
    * :kconfig:option:`CONFIG_MPSL_CX`
    * :kconfig:option:`CONFIG_MPSL_CX_3WIRE`
 
+.. _ug_radio_mpsl_cx_generic_1wire:
+
+Generic one wire coexistence
+============================
+
+Refer to :ref:`ug_radio_coex_mpsl_cx_based` for the general requirements of this implementation.
+
+The generic one wire coexistence interface can be used to allow the :ref:`SoftDevice Controller <nrfxlib:softdevice_controller>` to coexist alongside an LTE device on a separate chip.
```

Comment:
Not sure we should mention SoftDevice Controller explicitly here as this interface should work fine for 802.15.4 as well.
Right @ankuns?

Reply:
Generally it should.

Reply:
Shall we just mention both SDC and 802.15.4 ? This line just gives example use-cases